docs/source/hardware/lowpriority.rst

```diff
@@ -0,0 +1,79 @@
+
+.. _partition_names:
+
+**********
+Low priority partition
+**********
+:sub:`Updated September, 2022`
+
+
+Introduction
+===================
+``lowpriority`` is a new partition on Discovery that allows the research community to use private resources on the HPC cluster when they are idle. 
+This new partition has hardware not otherwise available to the general research community and, in time, could double the resources available to NURC users. 
+This is a common practice in HPC clusters to optimize the use of idle private resources that consume power and cooling. 
```

Comment:
This is a common practice in HPC clusters, which is employed to optimize the use of private resources that consume power via (?) cooling when idle.

I'm not sure what does "consume power and cooling" mean here.

Reply:
@gshomo  could you clarify here if "power and cooling" is the right usage?

Reply:
@gshomo, let us know what you think and we can resolve this accordingly. Thanks.

Reply:
@manasvita, @ShobanaSekar, @gshomo, perhaps just omit "that consume power and cooling"?

Reply:
I agree with @dhml. So maybe just keep it as: 

This is a common practice in HPC clusters to optimize the use of idle private resources